e2e-tests/src/pages/ourhealth/page-base.ts

```diff
@@ -0,0 +1,141 @@
+import { expect, Locator, Page } from '@playwright/test'
+import { RegisterPageInterface } from 'src/models/register-page-interface'
+import Footer from 'src/page-components/footer'
+import Navbar from 'src/page-components/navbar'
+import Question from 'src/page-components/question'
+
+export default abstract class PageBase implements RegisterPageInterface {
```

Comment:
`PageBase` is abstract and extended by all page classes. Avoids duplicated code and makes page classes much more cleaner and readable.